lib/utils.js

```diff
@@ -22,11 +22,26 @@ function isDomElement(o) {
   return o instanceof HTMLElement || o && o.nodeType > 0;
 }
 
-function renderTemplate({template, templateHelpers, data}) {
+function renderTemplate({template, defaultTemplate, config, data}) {
   var hogan = require('hogan.js');
   var forEach = require('lodash/collection/forEach');
+  var options = config.options;
   var content;
 
+  if (template === null || typeof template === 'undefined') {
```

Comment:
```js
typeof template === 'undefined'
```
is only needed when the variable may not have been defined in the first place. Which is not the case here because it is a parameter of the function. I would suggest that instead : 
```js
template === undefined
```